### PR TITLE
fix: hide navbar and sidebar in fullscreen mode

### PR DESCRIPTION
**Pull request by Udit-takkar**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ckbhatia** [comment]
Please reference the related issue

File: ui/pages/_app.js [hunk 1]
```diff
@@ -138,6 +140,12 @@ class MesheryApp extends App {
     this.meshsyncEventsSubscriptionRef.current = meshSyncEventsSubscription;
   }
 
+  fullscreenchanged = () => {
```
Comment (Ckbhatia) [inline ui/pages/_app.js:143]: camelCase

Comment (leecalcote) [reply ui/pages/_app.js:143]: Thanks for this, @ckbhatia. 

camelCasing applied.

**Review by leecalcote — APPROVED** [review]
Good!

**Pull request merged**